dev-util/gammaray/gammaray-2.9.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT=0
+IUSE="3d bluetooth concurrent designer doc geolocation printsupport
+	script scxml svg test qml wayland webengine"
+
+RDEPEND="
+	$(add_qt_dep qtcore)
```

Comment:
Didn't I complain about using this horrible eclass already? So many subshell calls are going to have serious impact on metadata cache generation speed, and I see no apparent reason to use magic to generate dependency strings. Especially given that having the proper cat/pn would be much more readable here.

Reply:
If it's not an actual KDE hosted project, I agree it does not make much sense to inherit kde5-functions just for the dep convenience.

Reply:
Good I will remove this.